### PR TITLE
Publish C# embedding as a nuget on nuget.org

### PR DESCRIPTION
**Pull request by swift-kim**
The workflow is only run manually on demand (`workflow_dispatch`).

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+on: [workflow_dispatch]
+
+jobs:
+  nuget:
+    runs-on: ubuntu-latest
+    if: ${{ github.repository_owner == 'flutter-tizen' }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-dotnet@v1
```
Comment (WonyoungChoi) [inline .github/workflows/release.yml:11]: It is recommended to use the latest version.

```suggestion
      - uses: actions/checkout@v3
      - uses: actions/setup-dotnet@v2
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        with:
+          dotnet-version: '6.0.x'
+      - name: Build and push
+        run: |
+          cd embedding/csharp/Tizen.Flutter.Embedding
```
Comment (WonyoungChoi) [inline .github/workflows/release.yml:16]: How about using `working-directory` property?
```suggestion
      - name: Build and push
        working-directory: embedding/csharp/Tizen.Flutter.Embedding
        run: |
```

File: .github/workflows/release.yml (hunk 1 above)
Comment (swift-kim) [reply .github/workflows/release.yml]: Fixed. Thanks!

**Review by WonyoungChoi — APPROVED** [review]
LGTM :+1:

**Pull request merged**